force-app/main/default/objects/Product_Registry__mdt/fields/Action__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Action__c</fullName>
+    <description>Defines the specific Action or Category this registry refers to.</description>
+    <externalId>false</externalId>
+    <fieldManageability>DeveloperControlled</fieldManageability>
```

Comment:
If we create the field as DeveloperControlled it means it can only be changed via package upgrade. Which means new products will be added there via EDA upgrade and admins won't be able to create products on their own, right?

Reply:
I believe we want developer-controlled on this, but I would appreciate the behavior of a subscriber-created record being validated through 2gp builds. I believe that another package can add additional records there.

Reply:
ok, so leaving it DeveloperControlled. Thanks